AudioBar/AudioBar.swift

```diff
@@ -57,6 +58,10 @@ public struct AudioBar: Program {
         let seekInterval: TimeInterval
         let playbackDuration: TimeInterval
         let elapsedPlaybackTime: TimeInterval
+        let trackName: String?
```

Comment:
We're using optionals for these values, because we don't want our code to crash if a tag is missing.